doc(GTFS): profile name in api docs CUSTOM_KEYS

takb opened the pull request:
- Reason for change:
Old profile names were still used in the CUSTOM_KEYS references

takb committed:
doc(GTFS): profile name in api docs CUSTOM_KEYS

TheGreatRefrigerator commented:
i still found these, do they also need adjustment?:
- [ ] https://github.com/GIScience/openrouteservice/blob/master/openrouteservice/src/main/resources/resources/locales/ors_de-DE.resources#L94
- [ ] https://github.com/GIScience/openrouteservice/blob/master/openrouteservice/src/main/resources/resources/locales/ors_en-US.resources#L94
- [ ] https://github.com/GIScience/openrouteservice/blob/master/openrouteservice/src/main/java/org/heigit/ors/api/responses/routing/json/JSONLeg.java#L38
- [ ] https://github.com/GIScience/openrouteservice/blob/master/openrouteservice/src/main/java/org/heigit/ors/api/responses/routing/json/JSONLeg.java#L112


For the locales, only US and DE are added for the public transit, not sure if there are other languages should also have the `pt` object.

takb committed:
Merge branch 'master' into doc/fix-routerequest-customkeys-gtfs

takb commented:
I don't think we need to use the full profile name for these four strings you mention, the language file key is something internal and the leg type value for the responses only produce unnecessary bloat. 

> For the locales, only US and DE are added for the public transit, not sure if there are other languages should also have the `pt` object.

But you are right about this, I'll add untranslated values to the other language files. Thx for the heads-up!

takb committed:
fix(GTFS): add public transport instructions strings to all locale files

translations are available for German and Japanese, all other languages fall back to English.
